Src/Private/Get-AbrHRZRegisteredMachine.ps1

```diff
@@ -111,6 +111,38 @@ function Get-AbrHRZRegisteredMachine {
                             catch {
                                 Write-PscriboMessage -IsWarning $_.Exception.Message
                             }
+
+                            section -Style Heading3 'Others' {
```

Comment:
The Other section appears empty in my report because I do not have any registered machines.

![Screenshot from 2023-12-19 15-45-55](https://github.com/AsBuiltReport/AsBuiltReport.VMware.Horizon/assets/1002783/056e5eb3-b1ca-4fec-9c56-96e0c5801525)

A suggestion would be to add a condition that validates if the $RegisteredPhysicalMachines variable holds content so that the section is not shown if it has nothing to display.


```powershell
if ($RegisteredPhysicalMachines) {
    section -Style Heading3 'Others' {
        Paragraph "The following section details the RDS Hosts configuration for $($HVEnvironment.toUpper()) server."
        BlankLine
        $OutObj = @()
        foreach ($RegisteredPhysicalMachine in $RegisteredPhysicalMachines) {
            Write-PscriboMessage "Other Registerd Machines"
            $inObj = [ordered] @{
                'Name' = $RegisteredPhysicalMachines.MachineBase.name
                'DNS Name' = $RegisteredPhysicalMachines.MachineBase.DnsName
                'Description' = $RegisteredPhysicalMachines.MachineBase.Description
                'OperatingSystem' = $RegisteredPhysicalMachines.MachineBase.Description
            }

            $OutObj += [pscustomobject](ConvertTo-HashToYN $inObj)
        }

        if ($HealthCheck.RegisteredMachines.Status) {
            $OutObj | Where-Object { $_.'Status' -ne 'AVAILABLE'} | Set-Style -Style Warning
        }

        $TableParams = @{
            Name = "Other Registered Machines - $($HVEnvironment.toUpper())"
            List = $false
            ColumnWidths = 20, 20, 30, 30
        }

        if ($Report.ShowTableCaptions) {
            $TableParams['Caption'] = "- $($TableParams.Name)"
        }
    }
}
```